backend/cmd/update/main.go

```diff
@@ -0,0 +1,363 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/rpc"
+	"os"
+	"os/exec"
+	"runtime"
+
+	"github.com/artdarek/go-unzip"
+	log "github.com/sirupsen/logrus"
+)
+
+func init() {
+
+	// initialize update.log file and set log output to file
+	file, err := os.OpenFile(getDefaultDagFolderPath()+"/update.log", os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0666)
```

Comment:
```suggestion
	file, err := os.OpenFile(path.Join(getDefaultDagFolderPath(), "update.log"), os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0666)
```

https://golang.org/pkg/path/#Join